melodic/distribution.yaml

```diff
@@ -6887,6 +6887,21 @@ repositories:
       url: https://github.com/fetchrobotics/robot_controllers.git
       version: melodic-devel
     status: maintained
+  robot_indicator:
+    doc:
+      type: git
+      url: https://github.com/LucidOne/robot_indicator.git
```

Comment:
The package.xml still lists TODO for the `<license>` tag, and there is no top-level LICENSE file in the repository.  Please fix both of these, then we can go ahead with this. Thanks.